cptv/bitstream.py

```diff
@@ -39,6 +39,10 @@ def uint32(self):
 
     def uint64(self):
         return struct.unpack("<Q", self.bytes(8))[0]
+
+    def string(self, data_len):
+        formatString = str(data_len) + 's'
```

Comment:
In Python lower_case_with_underscores is the preferred variable naming style (so `format_string` in this case). The joys of jumping languages...